packages/now-cli/test/dev/integration.js

```diff
@@ -916,45 +989,52 @@ test('[now dev] temporary directory listing', async t => {
       await sleep(ms('1s'));
     }
   } finally {
-    dev.kill('SIGTERM');
+    await dev.kill('SIGTERM');
   }
 });
 
 test('[now dev] add a `package.json` to trigger `@now/static-build`', async t => {
   const directory = fixture('trigger-static-build');
-  const { dev, port } = await testFixture(directory);
 
-  try {
-    dev.unref();
+  await fs.unlink(path.join(directory, 'package.json')).catch(() => null);
 
+  await fs
```

Comment:
Maybe `index.txt` can be a variable to be shared between this line and line 1018 in the script.